src/rustzcash.rs

```diff
@@ -67,6 +71,39 @@ fn read_le(from: &[u8]) -> FrRepr {
     f
 }
 
+/// Reads an FsRepr from [u8] of length 32
+/// This will panic (abort) if length provided is
+/// not correct
+fn read_fs(from: &[u8]) -> FsRepr {
+    assert_eq!(from.len(), 32);
+
+    let mut f = <<Bls12 as JubjubEngine>::Fs as PrimeField>::Repr::default();
+    f.read_le(from).expect("length is 32 bytes");
+
+    f.as_mut().reverse();
+
+    f
+}
+
+/// Writes an FsRepr to [u8] of length 32
+fn write_fs(mut f: FsRepr, to: &mut [u8]) {
+    assert_eq!(to.len(), 32);
+
+    f.as_mut().reverse();
```

Comment:
This needed to be removed as well.

Reply:
Should the reverse() on line 80 also be removed?